src/js/pouchdb-node.js

```diff
@@ -94,10 +98,53 @@ gpii.pouch.node.loadDataIfNeeded = function (that) {
     }
 };
 
+/**
+ *
+ * Remove all content, close, and then remove the directory content for this database.
+ *
+ * @param that
+ * @param callback
+ * @returns {*}
+ */
+gpii.pouch.node.cleanPouch = function (that, callback) {
+    var promise = fluid.promise();
+    gpii.pouch.cleanPouch(that, function (cleanupMessage) {
+        if (!cleanupMessage.ok) {
+            promise.reject(cleanupMessage);
+        }
+        else {
+            that.close().then(function (closeError) {
+                if (closeError) {
+                    promise.reject(closeError);
+                }
+                else {
+                    if (that.options.removeDirOnCleanup) {
+                        rimraf(that.options.dbOptions.prefix, function (rmDirError) {
```

Comment:
This is an issue when using `gpii.pouch.harness.persistent` where `that.options.dbOptions.prefix` equals to the `baseDir` specified in that grade instead of the individual database directory.